internal/cli/opsmanager/backup/backup.go

```diff
@@ -34,6 +35,7 @@ func Builder() *cobra.Command {
 		CheckpointsBuilder(),
 		EnableBuilder(),
 		config.Builder(),
+		blockstore.Builder(),
```

Comment:
[nit] big thing here, according to spec this should be `mongocli ops-manager admin backup blockstore(s) lists`
notice the admin part on the command, also make sure that this command is only available for Ops Manager and not Cloud Manager

Reply:
ah okay, good call 👍